src/ucp/tag/eager_rcv.c

```diff
@@ -168,13 +168,20 @@ ucp_eager_common_middle_handler(ucp_worker_t *worker, void *data, size_t length,
 {
     ucp_eager_middle_hdr_t *hdr = data;
     ucp_recv_desc_t *rdesc      = NULL;
+    ucp_ep_h ep UCS_V_UNUSED;
```

Comment:
why need UCS_V_UNUSED?

Reply:
the ep is not actually used, the variable is needed just to reuse a macro